src/lib/openapi/spec/public-signup-tokens-schema.ts

```diff
@@ -11,6 +11,7 @@ export const publicSignupTokensSchema = {
     properties: {
         tokens: {
             type: 'array',
+            description: 'An array of all the public signup tokens',
```

Comment:
This property is missing an example. What does it look like?

Reply:
added